app/components/forms/wizard/basic-details-step.js

```diff
@@ -106,6 +106,10 @@ export default Component.extend(FormMixin, EventWizardMixin, {
     return filter(this.get('data.event.tickets').toArray(), ticket => ticket.get('type') === 'paid').length > 0;
   }),
 
+  hasDonationTickets: computed('data.event.tickets.@each.type', function() {
```

Comment:
`hasPaidTickets` is only used at one place, for this very purpose, instead of defining this new computed property, combine it with `hasPaidTickets`. This will also eliminate the use of `or` inside template.

Reply:
@CosmicCoder96 Did the change